sdk/storage/src/core/mod.rs

```diff
@@ -3,16 +3,19 @@ mod connection_string;
 mod connection_string_builder;
 mod copy_id;
 mod copy_progress;
+mod errors;
 mod into_azure_path;
 pub mod prelude;
 pub mod shared_access_signature;
 pub use self::connection_string::{ConnectionString, EndpointProtocol};
 pub use self::connection_string_builder::ConnectionStringBuilder;
 pub use self::into_azure_path::IntoAzurePath;
-use azure_core::errors::AzureError;
 use azure_core::headers::*;
 pub use copy_id::{copy_id_from_headers, CopyId};
 pub use copy_progress::CopyProgress;
+pub mod parsing_xml;
```

Comment:
I like the idea to move the XML helper functions to a separate module. 
Does it need to be public?

Reply:
It no longer needs to be. I'll make it `pub(crate)` for this PR.